crates/lang/ir/src/ir/attrs.rs

```diff
@@ -496,7 +516,7 @@ impl core::fmt::Display for AttributeArg {
             Self::Constructor => write!(f, "constructor"),
             Self::Payable => write!(f, "payable"),
             Self::Selector(selector) => {
-                write!(f, "selector = {:?}", selector.to_bytes())
+                write!(f, "selector = {}", selector)
```

Comment:
If ir::Selector already implements Display we should ideally just forward to its Display implementation using core::fmt::Display::fmt(&selector, f).